README.md

```diff
@@ -93,12 +93,6 @@ After installing the above dependencies, make sure the submodules are setup:
 git submodule update --init --recursive
 ```
 
-Similarly, you'll need to setup K's Rust dependencies:
-
-```sh
-./deps/k/llvm-backend/src/main/native/llvm-backend/install-rust
-```
-
 If you haven't already setup K's OCaml dependencies more recently than February 1, 2019, then you also need to setup the K OCaml dependencies:
```

Comment:
While we're at it, is this still relevant (February 1, 2019 part)?

Reply:
That was the date I wrote this text. Still kinda relevant, though we've switched to LLVM backend as default. But you still need to setup ocaml dependencies if you want to use the OCaml backend.